src/components/ConsoleIcons/index.tsx

```diff
@@ -33,6 +33,12 @@ export default function ConsoleLabel({name}): ReactElement {
           <ConsoleIconWrapper icon={ConsoleIcons.office} /> <b>Organization</b>
         </>
       );
+    case 'organizations':
```

Comment:
The organization icon is already added (right above this). Unless I'm not seeing it, I don't think there's an icon like this with the plural "Organizations" in Console.

Reply:
"Organizations" (plural) are visible in the console. Please see my comments https://github.com/aiven/aiven-docs/pull/440/files#r1745016554